frontend/src/lib/dataViewStrings.ts

```diff
@@ -0,0 +1,20 @@
+/* Copyright Contributors to the Open Cluster Management project */
+
+import { DataViewStrings } from '@stolostron/react-data-view'
+import { useMemo } from 'react'
+import { useTranslation } from './acm-i18next'
+
+export const useDataViewStrings = (): DataViewStrings => {
+  const { t } = useTranslation()
+  return useMemo(
+    () => ({
+      backLabel: t('Back'),
+      cancelLabel: t('Cancel'),
+      noItemsString: (plural: string) => t('No {{plural}} yet', { plural }),
+      getStartedMessage: (article: string, singular: string) =>
+        t('To get started, create {{article}} {{singular}}.', { article, singular }),
```

Comment:
We don't want to substitute translations into translations. I think you forgot to update this after we changed it in `react-data-view`. These two fields are both just strings.
<img width="341" alt="image" src="https://user-images.githubusercontent.com/42188127/222473218-60e1e027-629b-48a3-9b2b-bbf17eb71527.png">